astropy/visualization/lupton_rgb.py

```diff
@@ -0,0 +1,478 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Combine 3 images to produce a properly-scaled RGB image following Lupton et al. (2004).
+
+For details, see : http://adsabs.harvard.edu/abs/2004PASP..116..133L
+
+The three images must be aligned and have the same pixel scale and size.
+
+Example usage:
+    image_r = np.random.random((100,100))
+    image_g = np.random.random((100,100))
+    image_b = np.random.random((100,100))
+    image = lupton_rgb.make_lupton_rgb(image_r, image_g, image_b, filename='randoms.png')
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+from . import ZScaleInterval
+
+
+__all__ = ['make_lupton_rgb', 'Mapping', 'LinearMapping', 'AsinhMapping',
```

Comment:
That sounds good to me. The classes aren't that hard to use, but they're also quite specific use-cases that most people probably won't want.